docs/content/en/docs/triggers/message-queue-trigger/message-queue-trigger-kind-keda.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
We should move this to docs/triggers - so that it is parallel to `Message Queue Trigger` level itself.

![image](https://user-images.githubusercontent.com/1167948/88031368-20711b00-cb5a-11ea-9b1d-0e711ce21cd0.png)

Reply:
Done